core/src/main/java/org/geysermc/geyser/network/UpstreamPacketHandler.java

```diff
@@ -151,7 +151,7 @@ public PacketSignal handle(RequestNetworkSettingsPacket packet) {
         session.sendUpstreamPacketImmediately(responsePacket);
 
         session.getUpstream().getSession().setCompression(algorithm);
-        session.getUpstream().getSession().setCompressionLevel(this.geyser.getConfig().getBedrock().getCompressionLevel());
+        //session.getUpstream().getSession().setCompressionLevel(this.geyser.getConfig().getBedrock().getCompressionLevel());
```

Comment:
Should we remove this from the default config for now if we aren't going to use it?

Reply:
re-added in later commit, wont be needed